src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -97,7 +97,7 @@ public override async Task<bool> Execute()
             {
                 Logger.DebugLocalized(nameof(Resources.RetrievingManifest_Message), this.Id);
 
-                GitHub client = new GitHub(null, this.WingetRepoOwner, this.WingetRepo);
+                GitHub client = new GitHub(this.GitHubToken, this.WingetRepoOwner, this.WingetRepo);

```

Comment:
If the user passes in a bad token, this will not only block the update functionality, but also throw a Octokit.AuthorizationException: Bad credentials exception. Are you okay with that behavior, we should at least catch the exception and show the user a proper error message IMO

Reply:
Discussed offline. Current behavior in new, update, token, commands is to not catch this exception. This will just result in it getting thrown a little earlier if the user specifies a bad token

Reply:
Resolved offline, if user enters a bad token, we should notify them immediately with a friendly message